website/blog/2018-03-05-AWS-app-sync.md

```diff
@@ -94,7 +94,7 @@ return <S3Album track/>
 
 AWS Amplify favors a convention over configuration style of development, with a global initialization routine or initialization at the category level. The quickest way to get started is with an [aws-exports file](https://aws.amazon.com/blogs/mobile/enhanced-javascript-development-with-aws-mobile-hub/). However, developers can also use the library independently with existing resources.
 
-For a deep dive into the philosophy and to see a full demo, check out the video from [AWS re:Invent](https://www.youtube.com/watch?v=vAjf3lyjf8c).
+For a deep dive into the philosophy and to see a full demo, check out the video from [AWS re\:Invent](https://www.youtube.com/watch?v=vAjf3lyjf8c).
```

Comment:
Why does this need to be escaped?

Reply:
Docusaurus v3 now uses https://github.com/remarkjs/remark-directive to provide support for admonitions, and other things using syntax such as `:textDirective`, `::leafDirective` and `:::containerDirective`.

`Re:invent` is parsed as `Re` + `:invent` text directive.

This is what happens in the MDX playground if you turn the remark-directive option on:

<img width="1008" alt="CleanShot 2023-09-14 at 13 06 03@2x" src="https://github.com/facebook/react-native-website/assets/749374/a2ae5ba0-665b-4fb2-857a-ea1ee460f650">


This regression was captured by the visual regression tests:

![CleanShot 2023-09-14 at 12 51 24@2x](https://github.com/facebook/react-native-website/assets/749374/a066cc23-823d-403f-9e72-34945f70e3a6)

---

Technically it should probably be possible to add some code in Docusaurus v3 so that this escaping becomes un-necessary, and the unhandled AST directive nodes get serialized back as raw text. For now it's not implemented, and not even sure who should implement that 😅. Will think about it. I hope in the meantime it's not a blocker for you to merge this PR.